src/diagnostics/VideoTest.js

```diff
@@ -6,14 +6,14 @@ export default class VideoTest extends Test {
     super(...arguments);
     this.name = 'Video Test';
 
```

Comment:
The issue with this test was a race within `localMedia`. The `destroy()` method was calling `localMedia.stop()`, but the test completed so quickly that the stream hadn't been stored inside `localMedia` yet (because it was async).